pkg/scheduler/schedulercache/cache_test.go

```diff
@@ -575,6 +575,69 @@ func TestExpireAddUpdatePod(t *testing.T) {
 	}
 }
 
+func makePodWithEphemeralStorage(nodeName, ephemeralStorage string, ports []v1.ContainerPort) *v1.Pod {
```

Comment:
Could you please remove ports? We don't need to check ports in this test.

Reply:
done, ptal